blocks/api/raw-handling/image-corrector.js

```diff
@@ -57,4 +57,9 @@ export default function( node ) {
 	if ( node.height === 1 || node.width === 1 ) {
 		node.parentNode.removeChild( node );
 	}
+
+	// Restore emoji as images.
+	if ( node.className === 'emoji' ) {
```

Comment:
Does it make sense to also verify that there is a usable `node.alt` in the `if`?